README.md

```diff
@@ -1,2 +1,41 @@
-# Leaderoard
-this is the first project in week 4, module 2
+# Project Name
```

Comment:
### Optional
- [ ] It is important the project title shows the kind of project, for example `Shooter Game` instead of `Project Name`. Let us see our README file as our documentation of our code so that others can easily understand the project and code. The same applies to the description

Reply:
hi dear @addod19  yes i have used the readme file 
but what I'm doing is I'm merging from feature branch to development branch . 
so the development readme is empty but the new readme in the feature is full of information